### PR TITLE
Better static mode

### PR DESCRIPTION
**Pull request by Gusted**
- Use pre-defined static themes.
- Use local files to store static themes.

### CODE REVIEW

File: src/ui/devtools/components/body.tsx [hunk 1]
```diff
@@ -15,13 +15,8 @@ function Body({data, tab, actions}: BodyProps) {
     const previewButtonText = data.settings.previewNewDesign ? 'Switch to old design' : 'Preview new design';
     const {theme} = getCurrentThemePreset({data, tab, actions});
 
-    const wrapper = (theme.engine === ThemeEngines.staticTheme
+    const wrapper = (theme.engine === ThemeEngines.cssFilter || theme.engine === ThemeEngines.svgFilter 
         ? {
```
[the remark this answers is not in the document]
Comment (Gusted) [reply src/ui/devtools/components/body.tsx:19]: Static is being removed for now by dev tools as it's going to get a whole rewrite and going to work on local files etc. So for now it's going to be removed by static and in a further PR being added.

File: src/static/themes/variables.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
+:root {
```
Comment (erkinalp) [inline src/static/themes/variables.css:1]: also export them in dynamic mode

**Review by erkinalp — CHANGES_REQUESTED** [review]

File: src/static/themes/variables.css (hunk 2 above)
Comment (Gusted) [reply src/static/themes/variables.css:1]: Already being taking care of by #2767 [see the code](https://github.com/darkreader/darkreader/blob/master/src/inject/dynamic-theme/index.ts#L101)

Comment (erkinalp) [reply src/static/themes/variables.css:1]: those six aren't going to be enough after we get more themes than just dark and light

Comment (Gusted) [reply src/static/themes/variables.css:1]: Uhm, Their will be just dark and light however, the new presets can have different colors and they are being dynamically computed by the current in use preset.

**Comment by Gusted** [comment]
Doesn't seem must interest as well, so closing.

**Pull request closed without merging**